src/test/java/com/artipie/asto/lock/storage/StorageLockTest.java

```diff
@@ -56,13 +59,33 @@
     private final Key target = new Key.From("a/b/c");
 
     @Test
-    void shouldAddValueWhenAcquiredLock() {
+    void shouldAddEmptyValueWhenAcquiredLock() {
         final String uuid = UUID.randomUUID().toString();
-        new StorageLock(this.storage, this.target, uuid).acquire().toCompletableFuture().join();
+        new StorageLock(this.storage, this.target, uuid, Optional.empty())
+            .acquire()
+            .toCompletableFuture().join();
         MatcherAssert.assertThat(
-            this.storage.exists(new Key.From(new Proposals.RootKey(this.target), uuid))
+            this.storage.value(new Key.From(new Proposals.RootKey(this.target), uuid))
```

Comment:
@olegmoz you can use `byte[] b = new BlockingStorage(storage).value(key)` for tests